source/change-streams/tests/README.rst

```diff
@@ -158,4 +158,46 @@ The following tests have not yet been automated, but MUST still be tested
 #. The ``killCursors`` command sent during the "Resume Process" must not be allowed to throw an exception.
 #. ``$changeStream`` stage for ``ChangeStream`` against a server ``>=4.0`` that has not received any results yet MUST include a ``startAtOperationTime`` option when resuming a changestream.
 #. ``ChangeStream`` will resume after a ``killCursors`` command is issued for its child cursor.
-
+#. - For a ``ChangeStream`` under these conditions:
+      - Running against a server ``>=4.2``.
+      - The batch is empty or has been iterated to the last document.
+   - Expected result: 
+       - ``getResumeToken`` must return the ``postBatchResumeToken`` from the current command response.
+#. - For a ``ChangeStream`` under these conditions:
+      - Running against a server ``<4.2``.
+      - The batch is empty or has been iterated to the last document.
+   - Expected result: 
```

Comment:
I suspect driver's will fail this test against a 4.0.7+ server.  From Kevin's quote of Bernard:

> Batches returned by change streams from 4.0.7+ or 4.2 will always include a postBatchResumeToken field

Since a driver is likely to just look for a postBatchResumeToken field in the response regardless of server version, this test will likely fail, as the driver will behave exactly as it would when connected to a 4.2 server.  

But this is exactly what we want: since the server team backported the change to 4.0, drivers might as take advantage of this feature with a 4.0.7+ server as well.

@kevinAlbs what do you think?  Am I missing something or should we change the change streams spec to say that postBatchResumeToken is 4.0.7+?

Reply:
Good point. Since the server is backporting, I agree we should change the spec to say that postBatchResumeToken is 4.0.7+. For the sake of these tests, I think we can just change the conditions from ">=4.2" and "<4.2" to ">=4.0.7" and "<4.0.7". There are other references in the spec that reference postBatchResumeToken being introduced in 4.2. I can submit a PR to update those.

Reply:
Done

Reply:
@DmitryLukyanov, go ahead and make the change in change-streams.rst to go along with this change?  One less PR to review separately, and this will keep the two changes together.

Reply:
Done

Reply:
Changing 4.2+ to 4.07+ will only require us to check the server version in the test suite, correct? The implementation will still just pick up the field if it's available.

Reply:
Yeah, the implementation shouldn't do version checks, just check if the `postBatchResumeToken` is part of the response or not.